git-flow.md

```diff
@@ -57,6 +57,11 @@ requested. When your PR is approved, rearrange all the commits to have a descrip
 * [Another checklist with useful principles](https://dev.to/codemouse92/10-principles-of-a-good-code-review-2eg)
 * [Tips for being nice during a code review](https://developers.redhat.com/blog/2019/07/08/10-tips-for-reviewing-code-you-dont-like/)
 
+#### Pro tip 
```

Comment:
Pro tip is a bit general. Could you change it for "Referencing pull requests and issues in GitHub"?